<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSLQueryDialog</class>
 <widget class="QDialog" name="SSLQueryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSL Querier</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBar" name="toolBar">
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <addaction name="actionSearch"/>
     <addaction name="separator"/>
     <addaction name="actionAdd"/>
    </widget>
   </item>
   <item>
    <widget class="wProgressBar" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>2</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/Search_16x.svg</normaloff>:/Resource/icons/Search_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/AddWebService_16x.svg</normaloff>:/Resource/icons/AddWebService_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wProgressBar</class>
   <extends>QWidget</extends>
   <header location="global">w_progress_bar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../r.qrc"/>
 </resources>
 <connections/>
</ui>
